### DISCUSSION TITLE
Missing option: Sync Registrations to LDAP

### DISCUSSION THREAD

**Discussion by verymilan** [keycloak/keycloak#14572] [Archive]
Hi there, sorry if the category is wrong, wasn't sure about it. 

Anyway, We have a usecase, where one of our applications needs LDAP authentication. I read that there is supposed to be a "Sync Registrations"-option in the LDAP provider settings, but at least here, with the latest Keycloak release, it is missing.

How can i access this function? Because otherwise it appears to only sync new attributes of existing users to LDAP.

**Update:** I have spotted this option in the "base" theme and selected it. The option seems to be available now, but without any assets, only blank html everything now – even with unzipped theme jar :D ...anyway thats some progress, gonna test if it actually works asap.

**Comment by PedroDuarte536** [comment]
Was going through exacly the same problem yesterday (https://keycloak.discourse.group/t/ldap-sync-in-keycloak-19/17717), updated all the themes in my realm settings and nothing changed... Could you please give more insight on how you solved it?

Thank you :)

**Reply by verymilan** [reply]
i think i have selected the "base" theme, which caused no assets – even after trying to unzip the .jar file containing them. however, after navigating through the very "base" html, i discovered the missing option, selected it, switched the theme back to v2 and everything was as expected (except of an error when saving passwords, but it's already known and works anyway)

**Reply by PedroDuarte536** [reply]
Thank you so much! Just tried it out and it's working!

**Comment by RashKash103** [comment]
I've been trying to get this up and running but I haven't been able to find this option. How are you switching to the base theme? Switching it to `base` under the *Realm Settings* doesn't really seem to change anything.

Thanks!
